docs/stdlib/datetime.rst

```diff
@@ -101,35 +101,34 @@ Dates and Times
 
 .. _ref_std_datetime_intro:
 
-EdgeDB has two classes of date/time types:
+In EdgeDB, there are two types of date/time constructs:
 
 * a timezone-aware :eql:type:`std::datetime` type;
 
-* a set of "local" date/time objects, not attached to any particular
-  timezone: :eql:type:`cal::local_datetime`, :eql:type:`cal::local_date`,
-  and :eql:type:`cal::local_time`.
+* a set of "local" date/time types, not attached to any particular
+  timezone, such as :eql:type:`cal::local_datetime`,
+  :eql:type:`cal::local_date` and :eql:type:`cal::local_time`.
 
-There are also two different ways of measuring duration:
+There are two different ways of measuring duration:
 
-* :eql:type:`duration` using absolute and unambiguous units;
+* :eql:type:`duration` for using absolute and unambiguous units;
 
-* :eql:type:`cal::relative_duration` using fuzzy units like years,
-  months and days in addition to the absolute units.
+* :eql:type:`cal::relative_duration` for using fuzzy units like years,
+  months and days in addition to absolute units.
 
-All date/time operators and functions and type casts are designed to
-maintain a strict separation between timezone-aware and "local"
-date/time values.
+All referrable operators, functions and type casts are designed to maintain a
+strict separation between timezone-aware and "local" date/time values.
 
-EdgeDB stores and outputs timezone-aware values in UTC.
+EdgeDB store and output timezone-aware values in UTC format.
 
 .. note::
 
-    All the date/time types are restricted to years between 1 and
-    9999, including the end points.
+    All date/time types are stricted to years between 1 and 999, including
+    their end points.
```

Comment:
This reads a bit awkward to me: the endpoints belonging to `the years between 1 and 9999`. Feels a bit unnatural to state it this way.
```suggestion
    the years 1 and 9999.
```